### ISSUE TITLE
Auto restart after "max time error" or any other errors

### ISSUE THREAD

**Issue by Digigreg** [social-manager-tools/social-manager-tools#35]
### Versions

Replace the values below with your own:

- **Bot Version:** v0.5.5
- **Operating System:** macOS 10.14 Mojave

### Expected Behavior

Please describe the program's expected behavior.

The program works very well, I just write for a possible enhancement:
sometimes the Instagram Bot (that I'm using through the GUI) stops to work if the server response needs to much time (for example 30.000 ms). Then the bot shouts down itself.
Would be nice to have an option to let the bot restart/resume automatically after every kind of error, so it can be left to work 24/7, as it sleeps automatically at night.
Summing up I mean to have an automated-non-stop bot, which doesn't need human hands from some days.
Anyway it is not an issue, just a feature request.
Really great job!

### Actual Behavior

Sometimes "Instagram bot" from the GUI stops to work if the server response needs to much time.

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

Opend the GUI, select "Instagram bot" and turn it on. Then wait some hours randomly.

**Comment by ptkdev** [comment]
P.s. you use mac, mac go to sleep after 2h of no use... Go to:
settings --> battery --> power supply (second tab) and:
1. enable wifi flag if mac with sleep. 
2. disable mac stop if monitor is power off
